### DISCUSSION TITLE
Using PHP with Vercel

### DISCUSSION THREAD

**Discussion by reallukedev** [vercel/vercel#6744] [Help]
Hello!

I am trying to upload a website with PHP to Vercel using VercelPHP (https://github.com/juicyfx/vercel-php) but, when trying to build, I get an error on my vercel.php file. The error is: "Error: The pattern "*.php" defined in `functions` doesn't match any Serverless Functions inside the `api` directory. Learn More: https://vercel.link/unmatched-function-pattern" and, the vercel.json file looks like this:
```json
{
    "functions": {
        "api/*.php": {
            "runtime": "vercel-php@0.4.0"
        },
        "*.php": {
            "runtime": "vercel-php@0.4.0"
        },
        "dashboard/*.php": {
            "runtime": "vercel-php@0.4.0"
        },
        "api/admin/*.php": {
            "runtime": "vercel-php@0.4.0"
        }
    }
}
```

Thank you for taking the time to read this,
Reallukeisbest

**Comment by leerob** [comment]
Hey! Would you mind opening an issue on that repo instead?

**Reply by fu-sen** [reply]
Yup? Why did you give such an answer?
This should not be a problem specific to vercel-php, but a limitation of the functions you set in vercel.json:
https://vercel.com/docs/cli#project-configuration/functions
